compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -173,8 +173,8 @@ def __init__(
 
     @staticmethod
     def get_metadata(config: Config, conf_dir: pathlib.Path) -> dict:
-        # Piecemeal Config settings because for MT, most of the ST items are
-        # unrelated -- the MT (aka EndpointManager) does not execute tasks
+        # Piecemeal Config settings because for MU, most of the ST items are
```

Comment:
Sorry for not noticing earlier -- should "ST" convert to "SU" here?

```suggestion
        # Piecemeal Config settings because for MU, most of the SU items are
```

Reply:
Oh yeah, another good catch. Fortunately this is the only instance of ST in the codebase (in reference to single-tenant).